scripts/WinInstallFiles/InstallJMRI.nsi

```diff
@@ -508,6 +512,14 @@ SectionGroup "JMRI Core Files" SEC_CORE
     Delete "$OUTDIR\jh.jar"
     Delete "$OUTDIR\jdom-jdk11.jar"
  
+    ; -- Delete certain jython files that have been moved, as of JMRI 4.99.1
+    ; -- [List will increase as files are moved]
+    Delete "$OUTDIR\jython\SetMqttOptions.py"
```

Comment:
I don't understand how this is supposed to work for people who are happily using these now, having configured them into preferences, Logix, etc.

Could you describe that somewhere, please?  Maybe as a release note warning (in elp/en/releasenotes/current-draft-warnings.shtml ) or one of the script doc pages.

Reply:
Can always count on you to keep me honest: have put in warnings.  Note:  people may not be "happily" using at least one of the files listed  as it doesn't work (SetMqttOptions)!  The one in the jython/MQTT directory does work.  So this is helping to keep some people happy by deleting any local copies in jython/ directory .  [Note that I moved the MQTT example files in Git over a year ago and all references in JMRI help files already correctly point to them.]

Reply:
FYI:  I won't try this with any other example files as ramifications are large, as is being discussed in Developers' Forum.